src/printer.js

```diff
@@ -42,13 +43,17 @@ function getLast(arr) {
   return null;
 }
 
+function getNodeListProperty(node) {
```

Comment:
i think @czosel ended up adding like a `getBody` function that does something similar?

Reply:
@mgrip we can rename to `getBody`, do it?